packages/vscode-container-client/src/clients/DockerClientBase/DockerClientBase.ts

```diff
@@ -518,8 +519,17 @@ export abstract class DockerClientBase extends ConfigurableClient implements ICo
         output: string,
         strict: boolean,
     ): Promise<PruneImagesItem> {
-        // TODO: Parse output for prune info
-        return Promise.resolve({});
+        const deletedImageLineRegex = /^deleted: sha256:\s*(\w+)$/igm;
+
+        const deletedImages = parsePruneLikeOutput(output, {
+            spaceReclaimedRegex: undefined,

```

Comment:
All three calls to `parsePruneLikeOutput` have undefined for `spaceReclaimedRegex`, seems like you can just remove the option entirely.